README.md

```
@@ -1 +1,5 @@
-# 24th-Web-Team-1-BE
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/c970feac-c5a5-40f1-959d-3223be8d539d" width="150" height="150" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)